src/tests/patientScope/update_with_patient_scope/update_with_patient_scope.test.js

```diff
@@ -198,5 +202,29 @@ describe('Condition Tests', () => {
             expect(body.resourceType).toStrictEqual('OperationOutcome');
             expect(body.issue[0].details.text).toStrictEqual('The current patient scope and person id in the JWT token do not allow writing this resource.')
         });
+        test('Non patient resources can not be accessed with patient scopes', async () => {
+            const envValue = env.VALIDATE_SCHEMA;
+            env.VALIDATE_SCHEMA = '0';
+
+            const request = await createTestRequest();
+            const container = getTestContainer();
+            /**
+             * @type {import('../../../fhir/patientFilterManager').PatientFilterManager}
+             */
+            const patientFilterManager = container.patientFilterManager;
+            for (const resourceType of Object.values(COLLECTION)) {
+                if (!patientFilterManager.canAccessResourceWithPatientScope({ resourceType })) {
+                    const resp = await request
+                        .put(`/4_0_0/${resourceType}/1`)
+                        .send({ ...resourceStructure, resourceType })
+                        .set(getHeaders('patient/*.*'));
+
+                    if (resp.statusCode !== 404) {
```

Comment:
Let's create list of resouces that can be access or not. instead of this "if-case"

We should avoid if-else check from testcase where possible.